tests/business-logic/Users.test.ts

```diff
@@ -0,0 +1,84 @@
+import UserEntity from "@business-logic/User";
+
+import { hashPassword } from "@helpers/auth";
+import prisma from "@helpers/prisma";
+import { teardown } from "@helpers/tests/teardown";
+
+let hashedPassword:string;
+let userEmail = "lambert@gmail.com";
+
+describe("User",()=>{
+    beforeEach(async () => {
+        await teardown();
+        hashedPassword = await hashPassword("lambert12345");
+      });
+    
+      describe('Users Test',()=>{
+          it('should create new user',async()=>{
+              const userBody = {
+                  name:"Nsengimana Lambert",
+                  email:userEmail,
+                  password:hashedPassword
+              }
+              const user = new UserEntity()
+              const userCreation = await user.create(userBody)
+              const { message } = userCreation
+              expect(message).toBe("User created")
+          });
+          it('should avoid creating user when exists',async()=>{
```

Comment:
just a small thing and personal preference. can you add an empty paragraph between the it('...') blocks.

I also prefer paragraphs inside the blocks

```javascript
  it("should return an existing user", async () => {
    const company = await prisma.company.create({
      data: {
        name: "Academy Records",
      },
    });

    const user = await prisma.user.create({
      data: {
        name: "Chris Rock",
        email: "test2@nueno.com",
        password: passwordHash,
        companyId: company.id,
      },
    });

    const userEntity = new UserEntity();
    const result = await userEntity.find(user.id);

    expect(result?.name).toBe(user.name);
  }, 4000);
```

Reply:
Thank you for the feedback..i am going to correct the errors found